flake.nix

```diff
@@ -615,7 +615,7 @@
           type = "app";
           program = checkedShellScript system "plutarch-test-${name}"
             ''
-              cd ${self}/plutarch-test
+              { touch ./plutarch-test/.crumb 2> /dev/null && cd ./plutarch-test && rm .crumb ; } || cd ${self}/plutarch-test
```

Comment:
What is this?

Reply:
It's a workaround for a permission problem with `nix run`:
https://discord.com/channels/922576618424250418/922577570766143498/962077469765165076

Reply:
I'm honestly not sure what this workaround is supposed to do? Elaborate?

Reply:
The workaround code tests if directory `./plutarch-test/` is writable, and if so continues from there; if it doesn't it goes to `${self}/plutarch-test/` instead. I forget how exactly I ran into the problem, but `nix run .#test-ghc810-nodev` was failing because it didn't have permissions on the `${self}/plutarch-test` directory.